tests/epyccel/test_epyccel_mod.py

```diff
@@ -1,57 +1,111 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring/
+import sys
 from numpy.random import randint, uniform
 from numpy import isclose
 
 from pyccel.decorators import types
 from pyccel.epyccel import epyccel
 
+# Relative and absolute tolerances for array comparisons in the form
+# numpy.isclose(a, b, rtol, atol). Windows has larger round-off errors.
+if sys.platform == 'win32':
+    RTOL = 1e-13
+    ATOL = 1e-14
+else:
+    RTOL = 1e-14
+    ATOL = 1e-15
+
 def test_modulo_int_int(language):
     @types(int, int)
     def modulo_i_i(x, y):
         return x % y
 
     f = epyccel(modulo_i_i, language=language)
-    x = randint(1e6)
-    y = randint(low=1, high=100) # low=1 for avoid zero-division error
+    x = randint(0, 1e6)
+    nx = randint(-1e6, -1)
+    y = randint(low=-100, high=100)
+    y = 1 if y == 0 else y
+
 
-    assert f(x, y) == modulo_i_i(x, y)
-    assert isinstance(f(x, y), type(modulo_i_i(x, y)))
+    f_output = f(x, y)
+    modulo_i_i_output = modulo_i_i(x, y)
+    assert modulo_i_i(x, y)== f(x, y)
+    assert modulo_i_i_output == f_output
+    assert isinstance(f_output, type(modulo_i_i_output))
+
+    #test negative x
+    f_output = f(nx, y)
+    modulo_i_i_output = modulo_i_i(nx, y)
+    assert modulo_i_i_output == f_output
+    assert isinstance(f_output, type(modulo_i_i_output))
```

Comment:
To be certain that this works I think it would be best to have 4 tests:
```
modulo_i_i(x,y)
modulo_i_i(nx,y)
modulo_i_i(x,ny)
modulo_i_i(nx,ny)
```